pkg/scripts/common-functions.sh

```
@@ -0,0 +1,262 @@
+    ca-certificates debhelper dpkg-dev gnupg2 wget sudo lsb-release
+
+  echo "Configuring pgEdge repository..."
+  configure_pgedge_apt_repo $REPO_TYPE
+
+  install_syft
+}
+
+rename_ddeb_packages(){
+
+  BUILD_DIR=$1
+  pushd $BUILD_DIR
```

Shellcheck (0.11.0)

[warning] Use 'pushd ... || exit' or 'pushd ... || return' in case pushd fails.

(SC2164)